src/main/java/org/dinosaur/foodbowl/domain/store/api/StoreController.java

```diff
@@ -0,0 +1,42 @@
+package org.dinosaur.foodbowl.domain.store.api;
+
+import jakarta.validation.Valid;
+import java.net.URI;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.dinosaur.foodbowl.domain.store.dto.StoreRequest;
+import org.dinosaur.foodbowl.domain.store.dto.StoreResponse;
+import org.dinosaur.foodbowl.domain.store.application.StoreService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/stores")
```

Comment:
```suggestion
@RequiredArgsConstructor
@RequestMapping("/api/v1/stores")
@RestController
```
중요한 것을 아래로 두고 아래에서부터 읽어올 수 있게 하는건 어떨까요?